scripts/sdfw/requirements.txt

```diff
@@ -0,0 +1 @@
+pynrfjprog
```

Comment:
I assume that this is problematic. Is it acceptable to require this? And should it be in this file, or moved to the top level requirements?

Only a handful of users will ever interact with these scripts, so I feel its a bit too much to have it in the top level.

Reply:
I think we should ideally update the scripts to instead use `pylink-square`, which is already pulled in by zephyr:
https://github.com/nrfconnect/sdk-nrf/blob/89793427ef0e9a852b56d9d011ef3873e3c3ab1e/scripts/requirements-fixed.txt#L94

Reply:
Updated to use pylink.